src/main/java/seedu/duke/userinterface/InputParser.java

```diff
@@ -7,27 +7,25 @@
 import seedu.duke.exceptions.InvalidSectionException;
 import seedu.duke.exceptions.TaskTitleException;
 import seedu.duke.exceptions.TaskWrongFormatException;
-import seedu.duke.userinterface.command.Add;
-import seedu.duke.userinterface.command.AddTimetable;
 import seedu.duke.userinterface.command.CliCommand;
 import seedu.duke.userinterface.command.Exit;
 import seedu.duke.userinterface.command.Help;
-import seedu.duke.userinterface.command.List;
-import seedu.duke.userinterface.command.ListTimetable;
 import seedu.duke.userinterface.command.ModeSwitch;
-import seedu.duke.userinterface.command.Remove;
-import seedu.duke.userinterface.command.RemoveTask;
-import seedu.duke.userinterface.command.Select;
+import seedu.duke.userinterface.command.notebook.AddNotebookMode;
+import seedu.duke.userinterface.command.notebook.ListNotebookMode;
+import seedu.duke.userinterface.command.notebook.RemoveNotebookMode;
+import seedu.duke.userinterface.command.notebook.SelectNotebookMode;
+import seedu.duke.userinterface.command.timetable.AddTimetableMode;
+import seedu.duke.userinterface.command.timetable.ListTimetableMode;
+import seedu.duke.userinterface.command.timetable.RemoveTimetableMode;
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
 
-import static seedu.duke.userinterface.command.AddTimetable.DEADLINE_DELIMITER;
-import static seedu.duke.userinterface.command.AddTimetable.TASK_DELIMITER;
-import static seedu.duke.userinterface.command.Select.NOTEBOOK_DELIMITER;
-import static seedu.duke.userinterface.command.Select.PAGE_DELIMITER;
-import static seedu.duke.userinterface.command.Select.SECTION_DELIMITER;
+import static seedu.duke.userinterface.command.notebook.SelectNotebookMode.*;
```

Comment:
Probably should not have wildcard imports...

Reply:
I blame IntelliJ for automatically "optimising" that